src/dmd/root/outbuffer.d

```diff
@@ -151,7 +157,6 @@ struct OutBuffer
         offset += len;
     }
 
-    // write newline
```

Comment:
Unrelated?

Reply:
it has to do with documenting the indent ability.
which won't work if you don't use `writenl`